modules/apps/digital-signature/digital-signature-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -155,4 +155,18 @@
 			max-height: 100%;
 		}
 	}
+
+	.file-entries-list .has-error {
```

Comment:
This class should be placed before `.envelope-list`

To respect the alphabetical order